src/main/java/com/google/sps/servlets/NotificationServlet.java

```diff
@@ -0,0 +1,209 @@
+                                                               Utility.SQL_LOCAL_USER,
+                                                               Utility.SQL_LOCAL_PASSWORD);
+          PreparedStatement pst = connection.prepareStatement(query);
+          ResultSet rs = pst.executeQuery()) {
+        // Defines the url to which the user will be redirected. Will be defined each case because
+        // URL will change once notifications for approvals and meetings are enabled.
+        String elementUrl = "/questions.html?id=" + modifiedElementId;
+        // Insert notification and get its id to relate in UserNotification table.
+        insertToNotification(connection, "You got an answer", elementUrl, localTimestamp);
+        int notificationId = getNotificationId(connection, localTimestamp);
+        // Iterate through the query's result set to insert all notifications.
+        while (rs.next()) {
+          insertToUserNotification(connection, rs.getInt(1), notificationId);
+        }
+        // Close the connection once all insertions have been performed.
+        connection.close();
+      } catch (SQLException ex) {
+        Logger lgr = Logger.getLogger(DataServlet.class.getName());
+        lgr.log(Level.SEVERE, ex.getMessage(), ex);
+      }
+    } else if (typeOfNotification.equals("answer")) {
+      // If the notification is for a new comment in an answer.
+      String query =  "SELECT follower_id FROM AnswerFollower WHERE answer_id = " +
+                       modifiedElementId;
+      // Query the information from QuestionFollower table.
+      try (Connection connection = DriverManager.getConnection(Utility.SQL_LOCAL_URL,
+                                                               Utility.SQL_LOCAL_USER,
+                                                               Utility.SQL_LOCAL_PASSWORD);
+          PreparedStatement pst = connection.prepareStatement(query);
+          ResultSet rs = pst.executeQuery()) {
+        // Defines the url to which the user will be redirected. Will be defined each case because
+        // URL will change once notifications for approvals and meetings are enabled.
+        String elementUrl = "/questions.html?id=" + modifiedElementId;
+        // Insert notification and get its id to relate in UserNotification table.
+        insertToNotification(connection, "Somebody commented your answer", elementUrl, 
+                             localTimestamp);
+        int notificationId = getNotificationId(connection, localTimestamp);
+        // Iterate through the query's result set to insert all notifications.
+        while (rs.next()) {
+          insertToUserNotification(connection, rs.getInt(1), notificationId);
+        }
+        // Close the connection once all insertions have been performed.
+        connection.close();
+      } catch (SQLException ex) {
+          Logger lgr = Logger.getLogger(DataServlet.class.getName());
```

Comment:
Change DataServlet to NotificationServlet.

Reply:
Done!